hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupBasedLoadBalancer.java

```diff
@@ -77,6 +81,23 @@
   private volatile RSGroupInfoManager rsGroupInfoManager;
   private LoadBalancer internalBalancer;
 
+  /** Define the config key of if fallback group is enabled */
+  public static final String FALLBACK_GROUP_ENABLED_KEY = "hbase.rsgroup.fallback.groups.enable";
```

Comment:
This config doesn't need be introduced? Enable this when config FALLBACK_GROUPS_KEY.

Reply:
OK, I keep only FALLBACK_GROUPS_KEY and remove FALLBACK_GROUP_ENABLED_KEY.